src/Tizen.System.Session/Session/SessionStructs.cs

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using System.Text;
+using Tizen.Internals;
+
+namespace Tizen.System
+{
+    [NativeStruct("subsession_event_info", Include="sessiond.h", PkgConfig="libsessiond")]
+    [StructLayout(LayoutKind.Explicit)]
+    internal struct SubsessionEventInfoNative
+    {
+        [FieldOffset(0)]
+        public SessionEventType eventType;
+        [FieldOffset(4)]
+        public int sessionUID;
+
+        [FieldOffset(8)]
+        public IntPtr AddUserPtr;
```

Comment:
I hope to add some comment here,
because, Actually it is not used and not correct, 
so I concern someone use it.

Reply:
What about marking the fields as private and adding the comment that the actual work is done via `Marshal.PtrToStructure` and `Marshal.PtrToStringAnsi` calls? Removing those fields won't work as gbs performs additional native structure validation step.